tests/renderer/versions-spec.ts

```diff
@@ -181,12 +174,6 @@ describe('versions', () => {
       );
       mockFetchOnce(mockUnpkgResponse.toString());
 
-      // return whether or not version in JSON is >=0.24.0
-      (semver.gte as jest.Mock).mockReturnValueOnce(true);
-      (semver.gte as jest.Mock).mockReturnValueOnce(true);
-      (semver.gte as jest.Mock).mockReturnValueOnce(true);
-      (semver.gte as jest.Mock).mockReturnValueOnce(false);
-
```

Comment:
Note: there's no need to mock semver here because testing >= 0.24.0 is what the production code does too.